packages/autocomplete-preset-algolia/src/search/fetchAlgoliaResults.ts

```diff
@@ -35,6 +35,11 @@ export function fetchAlgoliaResults<TRecord>({
 }: SearchParams): Promise<
   Array<SearchResponse<TRecord> | SearchForFacetValuesResponse>
 > {
+  if (typeof searchClient !== 'object') {
+    throw new Error(
+      'The `searchClient` option must be an initialized search client.'
+    );
```

Comment:
This is a development mistake so why not use `invariant` here? It doesn't need to end up in the production bundle.